src/components/App.tsx

```diff
@@ -999,6 +1000,10 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     }
   }
 
+  private onScroll = debounce(() => {
+    this.setState({ ...this.getCanvasOffsets() });
+  }, 500);
```

Comment:
can we add this magic number to: `constants.ts`

Reply:
👍 probably we should have `time_constants` file for all time related constants

Reply:
Let's keep them all there now.. and we'll split them if gets out of hand.